docs/anomaly-detection/Overview.md

```diff
@@ -0,0 +1,296 @@
+---
+title: VictoriaMetrics Anomaly Overview
```

Comment:
as we already use `VictoriaMetrics Anomaly Detection` throughout the docs, maybe change title to simply "Overview"/"vmanomaly overview" or use full one "VictoriaMetrics Anomaly Detection overview" ?